packages/flutter_blue/README.md

```diff
@@ -0,0 +1,157 @@
+<br>
+<p align="center">
+<img alt="FlutterBlue" src="https://github.com/pauldemarco/flutter_blue/blob/master/site/flutterblue.png?raw=true" />
+</p>
+<br><br>
+
+## Introduction
+
+FlutterBlue is a bluetooth plugin for [Flutter](https://flutter.dev), a new app SDK to help developers build modern multi-platform apps.
+
+## Alpha version
+
+This library is actively developed alongside production apps, and the API will evolve as we continue our way to version 1.0.
```

Comment:
README file need not be copied. The following content seems enough.

# flutter_blue_tizen

## Usage

This package is not an _endorsed_ implementation of `flutter_blue`. Therefore, you have to include `flutter_blue_tizen` alongside `flutter_blue` as dependencies in your `pubspec.yaml` file.

```yaml
dependencies:
  flutter_blue: ^0.8.0
  flutter_blue_tizen: ^1.0.0

```

Then you can import `flutter_blue` in your Dart code:

```dart
import 'package:flutter_blue/flutter_blue.dart';
```

For detailed usage, see https://pub.dev/packages/flutter_blue#usage.

## Required privileges

To use this plugin in a Tizen application, you need to declare the following privileges in your `tizen-manifest.xml` file.

```xml
<privileges>
    <privilege>http://tizen.org/privilege/bluetooth</privilege>
</privileges>
```

## Supported devices

Plugin is supported on Tizen 6.5 iot-headed armv7l and aarch64 (RPI4, RB5).